### PR TITLE
Fix broken license link

### PR DESCRIPTION
**Pull request by mjclemente**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Use `true`/`false`
 
 ### How are my contributions licensed?
 
-Please see the [LICENSE](/foundeo/cfdocs/LICENSE) file.
+Please see the [LICENSE](https://github.com/foundeo/cfdocs/blob/master/LICENSE) file.
```
Comment (shaedrich) [inline CONTRIBUTING.md]: I guess, this might suffice.
```suggestion
Please see the [LICENSE](LICENSE) file.
```

Comment (mjclemente) [reply CONTRIBUTING.md]: I don't think that works when the rendered docs are presented on cfdocs.org. The result would take you to: https://cfdocs.org/license - which 404s.

Comment (mjclemente) [reply CONTRIBUTING.md]: I'll handle it however you like.

Comment (shaedrich) [reply CONTRIBUTING.md]: I guess, the way you did it, is better :+1:

Well, finally it's @pfreitag's decision. I'm just an ordinary contributor^^

**Comment by pfreitag** [comment]
Merged it - that works.

**Pull request merged**